### PR TITLE
WarpX: 22.08

### PR DESCRIPTION
**Pull request by ax3l**
Update `warpx` & `py-warpx` to the latest release, `22.08`.

### CODE REVIEW

**Review by dpgrote — DISMISSED** [review]
Looks good

**Comment by alalazo** [comment]
@spackbot style

**Comment by alalazo** [comment]
@spackbot help

**Comment by alalazo** [comment]
@spackbot fix style

**Review by tldahlgren — DISMISSED** [review]
Confirmed sha256.

File: var/spack/repos/builtin/packages/py-warpx/package.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ class PyWarpx(PythonPackage):
 
     # NOTE: if you update the versions here, also see warpx
     version("develop", branch="development")
+    version('22.08', sha256='5ff7fd628e8bf615c1107e6c51bc55926f3ef2a076985444b889d292fecf56d4')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-warpx/package.py]: Black requires:

```suggestion
    version("22.08", sha256='5ff7fd628e8bf615c1107e6c51bc55926f3ef2a076985444b889d292fecf56d4')
```

**Comment by alalazo** [comment]
@spackbot fix style

**Review by alalazo — APPROVED** [review]

**Review by alalazo — CHANGES_REQUESTED** [review]
WarpX has not been updated to v22.08

**Comment by ax3l** [comment]
Ouch, thank you so much for catching that :pray:

**Review by alalazo — APPROVED** [review]

**Pull request merged**